### DISCUSSION TITLE
old API authentication scheme usable

### DISCUSSION THREAD

**Discussion by pkoevesdi** [kimai/kimai#5122] [On-Premise: Installation & Updates]
The Kimai mobile App seems somehow broken: it seems to only work with the old API authentication X-AUTH-USER. On the demo installations it works, but not on our on-premise kimai. I don't ask for support for the app (the developers don't react on any channel I tried), but as a workaround to make it work until fixed: is it possible to switch on the old authentication scheme on our self-hosted kimai?

**Comment by kevinpapst** [comment]
It doesn't have to be turned on somehow, the old way of authentication is still available.
Its just hidden in the UI:
<img width="1195" alt="Bildschirmfoto 2024-10-20 um 20 48 38" src="https://github.com/user-attachments/assets/0ce97d73-a5b8-4865-ab67-b5b8c2342fb1">
